LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
@JFU-GIT This file should be used: https://github.com/CMSgov/cms-open-source-policy/blob/master/LICENSE